src/replication.thrift

```diff
@@ -478,6 +478,21 @@ struct query_disk_info_response
     4:list<disk_info> disk_infos;
 }
 
+// This request is sent from clent to replica_server.
+struct migrate_replica_request 
```

Comment:
```suggestion
struct replica_disk_migrate_request 
```

`migrate_replica` is ambiguous, it's in fact only works within disk-migration. The former name can easily misguide people that it's for general replica migration.